includes/REST_API/Stories_Controller.php

```diff
@@ -593,6 +646,20 @@ private function add_post_locking_link( array $links, WP_Post $post ): array {
 		return $links;
 	}
 
+	/**
+	 * Helper method to get publisher logo id.
+	 *
+	 * @since 1.21.0
+	 *
+	 * @param WP_Post $post Post Object.
+	 * @return int ID of attachment for publisher logo.
+	 */
+	private function get_publisher_logo_id( WP_Post $post ): int {
+		$publisher_logo_id = get_post_meta( $post->ID, Story_Post_Type::PUBLISHER_LOGO_META_KEY, true );
+
+		return is_numeric( $publisher_logo_id ) ? (int) $publisher_logo_id : 0;
```

Comment:
When is it not numeric? We're using `register_post_meta()` with a default value for this meta key, so just curious when this is ever the case

Reply:
There is a small possibility this might be either false and numeric string. But this mostly for linter here.